### ISSUE TITLE
Incorrect key codes in C-S bindings

### ISSUE THREAD

**Issue by hrdl-github** [ahrm/sioyek#1111]
I haven't tested this rigorously, but the default `<C-S-o>` binding seems to break on the development branch independent of the value of `use_legacy_keybindings`. Bindings to e.g. `<C-S-O>` seem to work, but this not reflected in the default configuration or documentation.

**Comment by ahrm** [comment]
Yes, I should update the keys.config file once the development branch is finalized.

**Comment by panangam** [comment]
`<C-S-O>` feels odd to me. So the possible key combinations are 
```
o, O, <C-o>, <C-S-O>
```
It's like you have to say "press shift" twice. Instead, `<C-S-o>` as before or `<C-O>` would make more sense, as there's no redundant information. So the possible combinations will be
```
o, O, <C-o>, <C-S-o>
```
as before, or
```
o, O, <C-o>, <C-O>
```
The last version looks the most uniform to me. I think I found the relevant code [here](https://github.com/ahrm/sioyek/blob/development/pdf_viewer/input.cpp#L7308-L7328), and I'd be happy to put in the work. But I wanna make sure this fits in with your vision.

**Comment by ahrm** [comment]
Thanks for the offer :). I would like to implement it myself because there might be a lot of edge cases on different platforms and a lot of testing should be done.

**Comment by panangam** [comment]
Very understandable 👍. As always, thank you for the good work you put into this.

**Comment by ahrm** [comment]
I have added it in https://github.com/ahrm/sioyek/commit/bc5e582e482614fe1b7714971dac771f257a3de0 by the way.

**Comment by panangam** [comment]
Works great for me.
